src/main/java/seatsio/events/BestAvailable.java

```diff
@@ -21,11 +22,12 @@ public BestAvailable(int number, List<String> categories) {
         this.categories = categories;
     }
 
-    public BestAvailable(int number, List<String> categories, List<Map<String, Object>> extraData, List<String> ticketTypes) {
+    public BestAvailable(int number, List<String> categories, List<Map<String, Object>> extraData, List<String> ticketTypes, Boolean tryToPreventOrphanSeats) {
```

Comment:
I would create a new, overloaded method and keep the current public API. 
That way we don't force everyone to think & make a decision about this flag when upgrading (it's an exotic case after all).

Reply:
Ok, I restored the constructor and added a builder - that's cleaner than having to pass in a bunch of null values for parameters you don't care about.